README.md

```diff
@@ -8,6 +8,12 @@ This repo contains Ansible code for OpenShift and Atomic Enterprise.
   ```
     dnf install -y ansible-1.9.4 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
   ```
+  - RHEL 7:
+  ```
+  yum install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  rpm --import http://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+  yum install ansible1.9-1.9.4-2.el7 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
```

Comment:
I would not mention specific ansible version since we have already `ansible-2.3.0.0-3.el7` in epel7.